policyengine_us/parameters/gov/states/ga/tax/income/subtractions/subtractions.yaml

```diff
@@ -1,13 +1,12 @@
 description: Colorado counts these sources as subtractions.
 values:
   2021-01-01:
-    - us_govt_interest
-    - ga_retirement_exclusion
-    - ga_military_retirement_exclusion
-    - taxable_social_security
-    - ga_investment_in_529_plan_deduction 
- 
-
+    - us_govt_interest  
```

Comment:
we need a list for 2021 **without**     - ga_military_retirement_exclusion

and then **the same** list but **with**  - ga_military_retirement_exclusion